thingif/src/main/java/com/kii/thingif/Alias.java

```diff
@@ -0,0 +1,4 @@
+package com.kii.thingif;
+
+public abstract class Alias {
```

Comment:
Should require `Parcelable`. So I think it should be abstract class instead of interface